sdk/core/azure-core-http-okhttp/pom.xml

```diff
@@ -92,6 +92,12 @@
       <groupId>com.squareup.okhttp3</groupId>
       <artifactId>okhttp</artifactId>
       <version>4.12.0</version> <!-- {x-version-update;com.squareup.okhttp3:okhttp;external_dependency} -->
+      <exclusions>
+        <exclusion>
+          <groupId>com.squareup.okio</groupId>
+          <artifactId>okio</artifactId>
```

Comment:
Where's the other okio module coming from?

Reply:
`com.squareup.okio:okio` and `com.squareup.okio:okio-jvm` both use the module name `okio`. Since we're using `okio-jvm` this can be excluded.